src/dlsnode/app/config/PerformanceConfig.d

```diff
@@ -49,5 +49,15 @@ public class PerformanceConfig
     **************************************************************************/
 
     uint number_of_thread_workers = 20;
+
+    /***************************************************************************
+
+        Disable Nagle's algorithm for neo sockets. This is important for
+        testing requests with large volumes of data -- Nagle's algorithm slows
+        them down too much, if left enabled.
+
+    ***************************************************************************/
+
+    bool no_delay = false;
```

Comment:
Wait, doesn't `no_delay = false` mean "Nagle's algorithm on"? (Mismatch between comment and code.)

Reply:
Right - isn't that what is says: "Disable Nagle's algorithm for neo sockets"?

Reply:
No:
* Nagle's algorithm disabled == `no_delay = true`.
* Nagle's algorithm enabled == `no_delay = false`.

(Or am I going crazy?)

Reply:
You're not going crazy, perhaps I'm missing the point :-). When one sets `no_delay` to `true` we do what the documentation says: `Disable Nagle's algorithm...`, right?

I'll just remove this field ;-).

Reply:
There ;-).